src/components/PrimaryPanes/Sources.css

```diff
@@ -160,33 +161,41 @@
 
 .source-outline-tabs .tab {
   background-color: var(--theme-toolbar-background);
-  border-bottom: 1px solid transparent;
-  border-color: var(--theme-splitter-color);
-  color: var(--theme-body-color);
+  border-top: 2px solid transparent;
+  border-bottom: 1px solid var(--theme-splitter-color);
+  color: var(--theme-toolbar-color);
   cursor: default;
   display: inline-flex;
   flex: 1;
   justify-content: center;
   margin-bottom: 0px;
   margin-top: -1px;
   overflow: hidden;
-  padding: 8px 8px 7px 8px;
+  padding: 5px 8px 7px 8px;
   position: relative;
   transition: all 0.25s ease;
 }
 
-.source-outline-tabs .tab:first-child {
-  border-inline-end: 1px solid var(--theme-splitter-color);
+.source-outline-tabs .tab:not(.active):hover {
+  background-color: var(--theme-toolbar-hover);
+  border-top: 2px solid rgba(0, 0, 0, 0.2);
 }
 
-.source-outline-tabs .tab:hover {
-  background-color: var(--theme-toolbar-background-alt);
-  border-color: var(--theme-splitter-color);
+.theme-dark .source-outline-tabs .tab:hover {
+  border-top: 2px solid var(--tab-line-hover-color);
 }
 
 .source-outline-tabs .tab.active {
-  background-color: var(--theme-body-background);
-  border-bottom-color: transparent;
+  border-top: 2px solid rgb(10, 132, 255);
+  color: rgb(0, 96, 223);
+}
+
+.theme-dark .source-outline-tabs .tab.active {
+  color: var(--theme-toolbar-selected-color);
+}
+
+.theme-dark .source-outline-tabs .tab.active:hover {
+  border-top: 2px solid rgb(10, 132, 255);
```

Comment:
We should not change box-model metrics (border-top-width, here) for a specific theme (`.theme-dark`). Themes should only change colors, and if possible we should change colors globally. Something like:

```css
:root {
  --source-tab-hover-border-color: pink;
  --source-tab-active-border-color: red;
}
:root.theme-dark {
  --source-tab-hover-border-color: green;
  --source-tab-active-border-color: lime;
}
```

Reply:
Ah I did not know that, thanks for letting me know! And to your other comments as well, thank you 👍